docs/public/components/creating-bundles.md

```diff
@@ -0,0 +1,48 @@
+# Creating a bundle
+
+A bundle is a grouping of related components. The goal of a bundle is to ease the usage of multiple components working together to constitute a product.
+
+One example is `DogStatsD`, a server to receive metrics locally from customer apps. `DogStatsD` is composed of 9+
+components, but most binaries just want to include `DogStatsD` as a whole.
+
+For use cases like that of DogStatsD, create a bundle.
```

Comment:
We should also explain what should be in a bundle (sub components) and what should not be in a bundle.